include/utils.h

```diff
@@ -99,8 +103,10 @@ inline int getRandomUniformInt(unsigned long int seed = 0, int min = 0,
  */
 inline float getRandomUniformFloat(unsigned long int seed = 0, float min = 0,
```

Comment:
I think we should use Time(0) here instead of 0 in default value of the generator ! Thus the default way of using this function is a real random generator between 0 and 1 ! (same for the function above for int)

Reply:
This is still the same conceptually, std::mt19937 gen(seed); take as input a number and gives an integer.

Reply:
I know this is from a previous PR, but we should not set the seed to a default - constant - value, this has to be changed if we want to have a random behaviour "by default"